permissions/component-mock-javamail.yml

```
@@ -0,0 +1,7 @@
+---
+name: "mock-javamail"
+github: "jenkinsci/mock-javamail"
```

@timja Feel free to rename this to `jenkinsci/lib-mock-javamail` if you want it to be consistent with the other libraries we host.

```suggestion
github: "jenkinsci/lib-mock-javamail"
```